volume-providers/vpc/messages/messages_en.go

```diff
@@ -184,6 +184,13 @@ var messagesEn = map[string]util.Message{
 		RC:          500,
 		Action:      "Wait for volume deletion",
 	},
+	"ListVolumesFailed": util.Message{
```

Comment:
change `ListVolumesFailed` to `ListVolumeFailed `

Reply:
Sure..